fbw-a32nx/src/wasm/fbw_a320/src/interface/SimConnectInterface.cpp

```diff
@@ -2075,27 +2078,37 @@ void SimConnectInterface::processEventWithOneParam(const DWORD eventId, const DW
     }
 
     // TODO Unsync EFIS baro
+    case Events::KOHLSMANN_INC:
     case Events::A32NX_FCU_EFIS_L_BARO_INC: {
       fcuEfisPanelInputs[0].baro_knob.turns = 1;
       fcuEfisPanelInputs[1].baro_knob.turns = 1;
       std::cout << "WASM: event triggered: A32NX_FCU_EFIS_L_BARO_INC" << std::endl;
       break;
     }
 
+    case Events::KOHLSMANN_DEC:
     case Events::A32NX_FCU_EFIS_L_BARO_DEC: {
       fcuEfisPanelInputs[0].baro_knob.turns = -1;
       fcuEfisPanelInputs[1].baro_knob.turns = -1;
       std::cout << "WASM: event triggered: A32NX_FCU_EFIS_L_BARO_DEC" << std::endl;
       break;
     }
 
+    case Events::KOHLSMANN_SET:
```

Comment:
Beware that the params for this event are:
param0: hpa * 16
param1: index of the altimeter to set (you should definitely check that param)

Does that match A32NX_FCU_EFIS_L_BARO_SET?

Reply:
`Does that match A32NX_FCU_EFIS_L_BARO_SET?`
Good point, no it doesn't. Maybe it would be a good idea to change the format of the custom event to also have two parameters, and the INC/DEC events with one parameter, instead of specifying the side in the event name

Reply:
This should be fixed now